### ISSUE TITLE
"password field is required" error when autofilling with password managers on mobile browsers

### ISSUE THREAD

**Issue by FillingTheVoidOnYT** [Goldfish-Social/Goldfish#19]
The "password field is required" error happens when my password manager bitwarden autofills the contents. My way around this has always been pressing space in the password bar then backspacing so the website thinks, I typed it but I decided to create an issue as many users won't know that little lifehack. 

Screenshot: 
![signal-2022-08-19-085931_002](https://user-images.githubusercontent.com/110159004/185660277-b6189c8b-11d4-4418-b901-63d2b3f5d579.png)

System:
![Screenshot from 2022-08-19 09-05-07](https://user-images.githubusercontent.com/110159004/185660534-089dbdf4-c288-4ef0-94e9-72f8411f9e18.png)

**Comment by mstdn** [comment]
Good catch! For the auto-fill on the password field on login we use 'current-password' but I'm not sure what Bitwarden uses for that. I'm gonna do some searching and otherwise ask them👌🏻

**Comment by FillingTheVoidOnYT** [comment]
> Good catch! For the auto-fill on the password field on login we use 'current-password' but I'm not sure what Bitwarden uses for that. I'm gonna do some searching and otherwise ask them👌🏻

https://bitwarden.com/help/public-api/
